src/main/java/bio/overture/ego/model/dto/UpdateUserRequest.java

```diff
@@ -31,6 +32,8 @@
   private UserType type;
   private StatusType status;
   private LanguageType preferredLanguage;
-  @NonNull private ProviderType providerType;
-  @NonNull private String providerSubjectId;
+  private String firstName;
+  private String lastName;
+  @NotNull private ProviderType providerType;
```

Comment:
Do we want to be able to change provider information via REST API?

Reply:
I included the provider info here to add an extra validation on the user's provider sub id & type when updating:
https://github.com/overture-stack/ego/blob/8bd699b784bb442458239f150cb073450e3f1521/src/main/java/bio/overture/ego/service/UserService.java#L511
It was my intention to not allow the provider info to be changed, the request would error out if the incoming user provider info didn't match our stored user. But maybe I missed something?